src/util/pointer.h

```diff
@@ -36,10 +36,12 @@ class Buffer
 	Buffer(unsigned int size)
 	{
 		m_size = size;
-		if(size != 0)
+		if(size != 0) {
 			data = new T[size];
-		else
+		}
+		else {
```

Comment:
There's no explicit example in https://dev.luanti.org/engine-dev-process/code-style-guidelines/ but please use the following notation if you're already touching these lines:
```suggestion
		} else {
```

Reply:
https://github.com/minetest/dev.luanti.org/commit/09277ae238109cf7dfabb5cd8df9db687da15a88